src/pages/designing/index.mdx

```diff
@@ -59,20 +66,58 @@ IBMers can learn more about accessing design software in the
 #### Sketch kit
 
 For previous users of the Expressive Sketch kit, please note that this is no
-longer being maintained. We've been working on a new Sketch kit with Carbon for
-IBM.com components, and the preliminary version is available on the
-[Sketch kit](designing/sketch-kit/) page. This version includes color tokens,
+longer being maintained. We have a new [Sketch kit](designing/sketch-kit/) that includes color tokens,
 and expressive and productive type styles, and some key Carbon for IBM.com
 components. We are continuing to add more components and you will receive
 notifications for any library updates within Sketch.
 
 #### Figma kit
 
-We also have a new Figma kit available. This preliminary kit includes all themes
-in one, but we are working on breaking the kit into separate themes, and we are
-also building Carbon for IBM.com components for use in your Figma designs.
+We also have a [Figma kit](designing/figma-kit) available. The kit includes libraries for each of the themes (White, Gray 10, Gray 90, and Gray 100) and includes color tokens and expressive and productive type styles. It also includes some preliminary components and we are building out more Carbon for IBM.com components each week.
+
+## Learn about the foundations
+
+### The Carbon ecosystem
+
+When you hear about people talking about Carbon, it’s important to know that it’s more than just one website.
+
+The following diagram shows the relationships of assets and guidance across IBM—from the foundations of Brand and the IBM Design Language, to the digital implementation in the Carbon Design System that in turn provides the foundation for our products and web pages.
```

Comment:
should "relationships" be singular?

Reply:
> should "relationships" be singular?

@JBrandwine I think I'll remove relationships altogether. Great catch!